pkg/apis/integreatly/v1alpha1/types/types.go

```diff
@@ -18,12 +20,23 @@ type SecretRef struct {
 	Namespace string `json:"namespace,omitempty"`
 }
 
+type Env struct {
+	Name  string `json:"name"`
+	Value string `json:"value"`
+}
+
+type Labels struct {
+	ProductName string `json:"productname"`
```

Comment:
Is it possible to make `labels` a flexible map of strings? Product name is quite specific to integreatly and it would be nice to be able to add custom labels

Reply:
I like your thinking, truth is I don't know, but more that likely its possible.

Reply:
sdk gave me an API rule violation when tried to do it , so will leave as is